### PR TITLE
Add Privacy Considerations section

### LINKED ISSUE #10: Privacy considerations?
I see there is a Security considerations section, but nothing touches on Privacy. It would be great to explain why y'all think this is fine to expose to the web (and point out that CSS Media Queries already does so).

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #10

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/user-preference-media-features-headers/pull/11.html" title="Last updated on Jul 25, 2023, 12:06 PM UTC (9eb72eb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/user-preference-media-features-headers/11/b136e69...9eb72eb.html" title="Last updated on Jul 25, 2023, 12:06 PM UTC (9eb72eb)">Diff</a>

### CODE REVIEW

**Comment by tomayac** [comment]
FYI @ miketaylr.

File: index.bs [hunk 1]
```diff
@@ -198,6 +198,15 @@ This document extends the [=policy-controlled client hints features=] with the f
 - <code><dfn export>ch-prefers-forced-colors</dfn></code> which has a [=default allowlist=] of `'self'`
 - <code><dfn export>ch-prefers-color-scheme</dfn></code> which has a [=default allowlist=] of `'self'`
 
+<h2 id="privacy-considerations">Privacy Considerations</h2>
+
+The User Preference Media Features Client Hint header exposes data at the HTTP
+request level that is otherwise attainable and/or detectable using JavaScript
```
Comment (beaufortfrancois) [inline index.bs:204]: One could argue that users can disable JavaScript. CSS is harder on the other hand ;)

File: index.bs [hunk 2]
```diff
@@ -198,6 +198,15 @@ This document extends the [=policy-controlled client hints features=] with the f
...
+and CSS as defined in [[!MEDIA-QUERIES-LEVEL-5]]. By providing this data at
+the request level, the server gains the ability to customize the response
+based on the user's preferences. Importantly, it does not grant the server
+access to any previously inaccessible information.
```
Comment (beaufortfrancois) [inline index.bs:208]: I'm not sure how to phrase it but we could also say that users can override those values with DevTools & Extensions even if it's not CSS/JavaScript.

Comment (tomayac) [reply index.bs:208]: In Chrome they can, but the statement is not necessarily true in all browsers.

**Review by beaufortfrancois — APPROVED** [review]

**Pull request merged**